TidepoolOnboarding/View Models/OnboardingViewModel.swift

```diff
@@ -103,6 +121,26 @@ class OnboardingViewModel: ObservableObject, CGMManagerCreateNotifying, CGMManag
         }
     }
     
+    func updateLastAccessedDate() {
+        if shouldRestart {
+            restart()
+        }
+
+        self.lastAccessDate = Date()
+    }
+
+    private let lastAccessDurationMaximum: TimeInterval = .days(7)
```

Comment:
[nit] Maybe I'm mis-reading or mis-understanding here... is this because prescriptions expire after 7 days? If so, shouldn't this maybe be the responsibility of `Prescription`, then?

Reply:
This is not related to the prescription. This is a specific product request to restart onboarding if the user hasn't viewed it in 7 days. It is to prevent the user from "forgetting" what they read a month ago and continuing on with onboarding.

Reply:
^ maybe something to the effect of this explanation might be worth including in a code comment?  It wasn't obvious to me...